design/0001-frr.md

```diff
@@ -46,7 +46,7 @@ provides detailed logging features to aid debugging.
 * Provide a method to integrate arbitrary BGP implementations into MetalLB.
 * Support runtime changing of BGP implementation type.
 * Extend the functionality of the MetalLB BGP implementation in in order to
-  take advantage of the additional cababilities of FRR.
+  take advantage of the additional capabilities of FRR.
```

Comment:
Despite these being typos, I don't want to change the design proposals after merging them to keep the original state.
It's not public facing doc so I think it's fine.